mod.ts

```diff
@@ -34,14 +34,17 @@ async function handleKillPortWindows(port: number): Promise<number | null> {
   return pid;
 }
 
-async function getPidPortWindows(port: number): Promise<number> {
+async function getPidPortWindows(port: number, options: string): Promise<number> {
   const cmd = Deno.run({
     cmd: ["cmd", "/c", "netstat -a -n -o | findstr", `${port}`],
     stdout: "piped",
     stderr: "piped",
   });
   const output = new TextDecoder("utf-8").decode(await cmd.output());
-  return parseInt(output.trim().split(/[\s, ]+/)[4]);
+  if (options.toUpperCase() === output.trim().split(/[\s, ]+/)[0].toUpperCase()) {
+    return parseInt(output.trim().split(/[\s, ]+/)[4]);
+  }  
+  return 0;
```

Comment:
Pode retornar null caso não encontre o PID.  Eu sei que depois disso ele faz uma checagem ( _if (!pid)_) que vai cair mesmo que for 0 ou null, o que daria no mesmo, mas é um bug que percebi agora na verificação que eu fiz anteriormente, pois o PID 0 existe e pertence a um processo. Vou arrumar depois.